### DISCUSSION TITLE
Black screen on RG280V

### DISCUSSION THREAD

**Discussion by Mythrid12** [eduardofilo/RG350_adam_image#227] [Hardware problems?]
I've tried flashing my new RG280V but it only boots to a black backlit screen. I've tried a number of different SD cards and different versions of adam image. The folder structure is created on the 2nd SD card at boot so something is happening. I can also power it off using Start + A. 

Stock image and rogue CFW works absolutely fine. 

One step weirder... I have a buddy with a RG280V running adam 1.4. I can take both of his micro SD cards, put them into my RG280V, and I have the same issue. I've seen this issue around other places (reddit, discord) and no one seems to have a solution. Any ideas?

**Comment by neilswann80** [comment]
Just want to rule out the obvious... although I don't think this is the issue as you've tested working cards from your buddy's device...

BUT after flashing the image have you ran the `select_kernal.bat` and chosen `1`, as outlined [HERE](https://github.com/eduardofilo/RG350_adam_image#windows)

Sounds like a hardware/graphics issue, but strange as to why it would only affect Adam Image.  Perhaps @eduardofilo will have some suggestions.  As it's powering off with `START-A` it's clearly running the software, hopefully there'll be some log files @eduardofilo can direct you to.  Do the brightness shortcuts have any effect?  e.g.  `POWER+LEFT` and `POWER+RIGHT `

> Stock image and rogue CFW works absolutely fine.

When you say `stock image` do you mean the original OD or ODBeta?
If you haven't tested the latest ODBeta can you test that?  You can get the stock image and the ODBeta upgrade [HERE](https://retrogamecorps.com/2020/12/05/opendingux-beta-firmware-for-rg350-and-rg280-devices/#Install)

**Reply by Mythrid12** [reply]
Yep, no worries... I did run the .bat file and choose option 1. I hadn't thought to try the brightness shortcuts but those do work as well. The backlight gets brighter/dimmer but still just a black screen unfortunately. 

I'm running original OD but I'm going to try ODBeta now.

**Comment by eduardofilo** [comment]
Well, it is perfectly possible for a piece of hardware to work well with one system and not work or behave differently with another, since they may be governed differently. But if you have selected the correct console in the kernel selection script and/or if it doesn't work with your buddy's cards, it seems clear that your display has some kind of marginality and doesn't suit ODbeta's way of governing it. Think that this screen is probably tested at the Anbernic factory with a stock kernel.

**Reply by Mythrid12** [reply]
That's kind of the conclusion that reddit came to. Might just be a batch with screens that don't play well with Adam for whatever reason. I'm about to try the OD Beta image to see if I have any luck with that out of curiosity.

**Comment by Mythrid12** [comment]
Yep, same issue with ODBeta. I was able to flash a new card with the OD original image, booted to that no problem, ran the ODBeta update, rebooted, and now I'm back at the black screen. I guess I'm just lucky and got a special screen. Lucky me! :D 

I ordered another 280V from a different shop so if the replacement works I'll send this one back.

**Reply by neilswann80** [reply]
Sorry to hear that!  I hope your new 280V works perfectly and you get to enjoy the Adam Image

**Reply by Mythrid12** [reply]
Eh, it happens. Either way thanks for trying to help! Hopefully the replacement will be good to go. I've heard really good things about Adam Image so I'm excited to give it a shot! 

Thanks again!

**Comment by Mythrid12** [comment]
Just to give an update... Got the replacement unit in and it boots to the flashed SD no problem. Boxing the other up and shipping it back. This really is so much better than the stock OS. Thanks again!

**Reply by neilswann80** [reply]
Excellent news.  Happy Gaming!

**Comment by ebzero** [comment]


**Reply by neilswann80** [reply]
Definitely the exact same issue?  Have you confirmed your device doesn't boot with vanilla/default ODbeta rather than Adam Image?  And tried a different microsd card etc?

(Apologies if this seems like I'm second guessing you here... but after years of IT Support calls it's always best to double check the easy stuff before going down the rabbit hole of hardware returns 😞)

**Reply by eduardofilo** [reply]
And you have run the kernel selection script, right?

**Reply by neilswann80** [reply]
Any progress @ebzero?

**Reply by ebzero** [reply]
@neilswann80  no progress.  Unfortunately, I had to just give up on this device.

**Reply by neilswann80** [reply]
@ebzero Did you try any of the things we suggested?  Can you confirm you ran the kernel selection script after flashing the image and before inserting in the device?  Have you confirmed your device works with OpenDingux-Beta rather than the default OpenDingux?  You can get the stock OpenDingux image and the ODBeta upgrade [HERE](https://retrogamecorps.com/2020/12/05/opendingux-beta-firmware-for-rg350-and-rg280-devices/#Install)

**Reply by ebzero** [reply]
I definitely ran the kernel selection script.  The out-of-the-box firmware that came with the device also worked fine.  Unfortunately, I decided to get rid of the device so I can't investigate any further.

**Reply by neilswann80** [reply]
Sorry to hear that!   Did you opt for a different emulation device instead?  If so, what did you go for?

**Reply by ebzero** [reply]
Miyoo Mini!

**Reply by neilswann80** [reply]
Nice choice... I nearly bought one recently.  Lovely looking device and I love that it's the same dimensions as a pack of playing cards, so there are plenty of carrying cases that'll fit perfectly.

Happy gaming!

**Comment by djemile13** [comment]
Hi, I'm having the exact same issue with the RG350P. I have tried all Adam releases. I also have tried several SD cards, and I have ran the .bat. Same black screen at the beginning. I haven't tried the OD Beta, I will do that

**Reply by eduardofilo** [reply]
In your case, also try selecting the RG350M model in the system selection script, as there are many cases of RG350P's with the RG350M screen fitted at the factory.

**Reply by djemile13** [reply]
It worked! Thanks a lot!!! I selected RG350M and it worked!

**Comment by djemile13** [comment]
Hi, thanks a lot, i will try this evening and give you a feedback

On Mon, Oct 10, 2022 at 4:47 PM Eduardo Moreno ***@***.***>
wrote:

> In your case, also try selecting the RG350M model in the system selection
> script, as there are many cases of RG350P's with the RG350M screen fitted
> at the factory.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/eduardofilo/RG350_adam_image/discussions/227#discussioncomment-3839590>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3PBJRNEXNY3UF4ENKA27EDWCQM6DANCNFSM5Q5GLEZQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> .com>
>


-- 
Best regards,

Aimilios Mouratis
M: 6942463571
